run/hello-broken/.dockerignore

```diff
@@ -0,0 +1 @@
+vendor/
```

Comment:
I don't think this is needed if we're using modules?

Reply:
This prevents the vendor/ directory from being copied into the container. It is valuable to be very explicit in excluding local-generated code/files that would be redundant to what happens inside the container.

Reply:
I think you'd want vendor/ to be copied inside the container if you have vendor/.

Reply:
yep, -1 on ignoring vendor for me too